cli/fathom_web/commands/vectorize.py

```diff
@@ -54,6 +54,8 @@ def main(ruleset_file, samples_directory, fathom_fox_dir, fathom_trainees_dir, o
       * yarn
       * A FathomFox repository checkout
       * A Fathom Trainees repository checkout
+      * A copy of Firefox
+      * A geckoview driver downloaded and accessible in your PATH environment variable
```

Comment:
AFAIK, [GeckoView](https://wiki.mozilla.org/Mobile/GeckoView) is an Android library for Firefox. Can we just say "geckodriver"?

Reply:
Whoops, you are right. Thanks!